addons/editor-square-inputs/addon.json

```
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/dist/schema.json",
+  "name": "Square block text and color inputs",
+  "description": "Makes text and/or color inputs on blocks rectangular instead of round, like in Scratch 2.0.",
+  "credits": [
+    {
+      "name": "CST1229",
+      "link": "https://github.com/CST1229/"
+    }
+  ],
+  "tags": ["editor"],
```

I would consider this a theme even if it does also help differentiate text and number inputs. It also only effects blocks, so it should also have the code editor tag.
```suggestion
  "tags": ["editor", "codeEditor", "theme"],
```